WordPress/Classes/ViewRelated/Reader/ReaderHelpers.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+class ReaderHelpers {
+
+    public class func shareControllerForPost(post:ReaderPost) -> UIActivityViewController {
```

Comment:
Agreed!.  I think I should probably also change this to not accept a ReaderPost, and either make it a content provider or a list or args.  Will edit.